### DISCUSSION TITLE
Docker container in repository

### DISCUSSION THREAD

**Discussion by jgsuess** [oracle/macaron#215] [Q&A]
I would like to experiment with macaron and was wondering if there is a released container deployed in any repository? The instructions seem to suggest building the system yourself, but the `docker` directory seems to have a definition. If a container exists, the repo location on the home page would be a great idea. It accelerates adoption.

**Comment by behnazh-w** [comment]
@jgsuess Thanks for reaching out. We are currently working on [our first release ](https://github.com/oracle-samples/macaron/milestone/1), which will provide Macaron as a container. The instructions will be updated once in our first release.
